src/components/CareerFramework/Join.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import StyledLink from '../Common/StyledLink'
+import styled from 'styled-components'
+
+import { Grid, Row, Col } from '../grid'
+import { BodyPrimary, SectionTitle } from '../Typography'
+
+import Image from '../Common/Image'
+
+const JoinContentCol = styled(Col)`
+  padding-top: ${({ theme }) => theme.space[7]};
+  padding-bottom: ${({ theme }) => theme.space[7]};
+`
+
+const Join = props => {
+  const { title, ctaTitle, ctaUrl, content, image } = props
```

Comment:
Same comment as above
`const Join = ({ title, ctaTitle, ctaUrl, content, image }) =>  <Grid> \\ rest of the code`